google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/standard/AppEngineStandardDeploymentEditor.java

```diff
@@ -0,0 +1,244 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.intellij.appengine.cloud.standard;
+
+import com.google.cloud.tools.intellij.appengine.cloud.AppEngineApplicationInfoPanel;
+import com.google.cloud.tools.intellij.appengine.cloud.AppEngineDeployable;
+import com.google.cloud.tools.intellij.appengine.cloud.AppEngineDeploymentConfiguration;
+import com.google.cloud.tools.intellij.appengine.cloud.AppEngineEnvironment;
+import com.google.cloud.tools.intellij.appengine.cloud.AppEngineRuntimeInstance;
+import com.google.cloud.tools.intellij.appengine.cloud.flexible.UserSpecifiedPathDeploymentSource;
+import com.google.cloud.tools.intellij.appengine.project.AppEngineProjectService;
+import com.google.cloud.tools.intellij.appengine.sdk.CloudSdkService;
+import com.google.cloud.tools.intellij.appengine.sdk.CloudSdkValidationResult;
+import com.google.cloud.tools.intellij.login.CredentialedUser;
+import com.google.cloud.tools.intellij.resources.ProjectSelector;
+import com.google.cloud.tools.intellij.ui.BrowserOpeningHyperLinkListener;
+import com.google.cloud.tools.intellij.util.GctBundle;
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.base.Strings;
+
+import com.intellij.openapi.options.ConfigurationException;
+import com.intellij.openapi.options.SettingsEditor;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.HyperlinkLabel;
+import com.intellij.ui.components.JBTextField;
+import com.intellij.util.ui.tree.TreeModelAdapter;
+
+import org.apache.commons.lang.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Set;
+
+import javax.swing.JCheckBox;
+import javax.swing.JComponent;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.event.TreeModelEvent;
+
+/**
+ * Editor for an App Engine Deployment runtime configuration.
+ */
+public class AppEngineStandardDeploymentEditor extends
+    SettingsEditor<AppEngineDeploymentConfiguration> {
+  private JPanel editorPanel;
+  private JBTextField versionIdField;
+  private ProjectSelector projectSelector;
+  private JCheckBox promoteCheckbox;
+  private JCheckBox stopPreviousVersionCheckbox;
+  private HyperlinkLabel promoteInfoLabel;
+  private AppEngineApplicationInfoPanel applicationInfoPanel;
+  private JLabel serviceLabel;
+  private JLabel environmentLabel;
+  private HyperlinkLabel appEngineCostWarningLabel;
+
+  private Project project;
+  private AppEngineDeployable deploymentSource;
+
+  private static final boolean PROMOTE_DEFAULT = true;
+  private static final boolean STOP_PREVIOUS_VERSION_DEFAULT = true;
+
+  /**
+   * Initializes the UI components.
+   */
+  public AppEngineStandardDeploymentEditor(Project project,
```

Comment:
Just ran reformat with google-java-format. It didn't change much. I'm curious what you thought was so strange about the format?